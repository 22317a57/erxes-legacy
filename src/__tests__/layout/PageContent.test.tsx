import { mount, shallow } from 'enzyme';
import * as React from 'react';

import PageContent from '../../modules/layout/components/PageContent';

describe('PageContent component', () => {
  const defaultProps = {
    transparent: false
  };

  test('renders PageContent successfully', () => {
    shallow(<PageContent {...defaultProps} />);
  });

  test('renders successfully with default value', () => {
    const wrapper = mount(<PageContent {...defaultProps} />);
    const props = wrapper.props();

    expect(props).toMatchObject(defaultProps);
  });
});
