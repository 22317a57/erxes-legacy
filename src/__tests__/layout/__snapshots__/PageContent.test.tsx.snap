// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContent component snapshot matches 1`] = `
<section
  className="sc-bZQynM cjaOUa"
>
  <div
    className="sc-gzVnrw kQrJgW"
  />
</section>
`;
